### ISSUE TITLE
Dashboard: switching organizations, editing organization

### ISSUE THREAD

**Issue by maxvisser** [teamforus/forus#1435]
## Context/goal: 

Proposal 1: 
![image](https://user-images.githubusercontent.com/10818702/76887546-b9c5b300-6882-11ea-9bcb-94be94a3c113.png)

Feedback:
make cards smaller 1/3 of size
make them look diferent then fund card

**Comment by maxvisser** [comment]
Proposal 2:
![image](https://user-images.githubusercontent.com/10818702/76887810-1b861d00-6883-11ea-9348-6e89db1056b6.png)
dropdown like this in menu

**Comment by danrminds** [comment]
Proposal 3: 

It can be done by replacing the Forus logo in the top left of the dashboard, and display the current organisation there. And make a dropdown when clicking that in case you have more organisations.

**Comment by danrminds** [comment]
Proposal 4:

A dropdown like this on all pages.
![image_2020_03_17T17_50_51_355Z](https://user-images.githubusercontent.com/33207036/76943384-8e34de00-68ff-11ea-9658-fb6060ddd760.png)

**Comment by maxvisser** [comment]
yes something like proposal 4 is what I envisioned with proposal 2 but above menu or even replacing the logo.

**Comment by maxvisser** [comment]
I think there should be a 'Manage organizations' page with all organization listed and a add / edit button.
Valentins feedback was that switching between organizations and this manage organization page should be different aswell as the manage organization page blocks should look different than for example fund cards

**Comment by danrminds** [comment]
Yes, I also took a look in your prototype. For me - switching I think should be easy to be made.
And yes - I agree that the cards about the organisations should be somewhat different than the funds, as it might cause confusion. Maybe rethink a bit and position it on the same page (funds being attributed to the organisations themselves.

**Comment by maxvisser** [comment]
I did it as menu item in the prototype as alot of users will only have one organization so for them the 'Manage organization' page is more important then switching :)
